lib/pbench/agent/tool_meister_start.py

```diff
@@ -187,7 +187,6 @@ class ReturnCode(BaseReturnCode):
     """
 
     BADTOOLGROUP = 1
-    BADAGENTCONFIG = 2
```

Comment:
If I might request it, instead of just deleting this line, please replace it with a placeholder -- e.g., a comment indicating that the value is currently unused.  This will save future readers from wondering if it was inadvertently omitted, and it will help future developers to find the hole if they want to put a new status in it.